code-samples/creating-your-bot/command-deployment/index.js

```diff
@@ -30,7 +30,11 @@ client.on(Events.InteractionCreate, async interaction => {
 		await command.execute(interaction);
 	} catch (error) {
 		console.error(error);
-		await interaction.reply({ content: 'There was an error while executing this command!', ephemeral: true });
+		if(interaction.replied || interaction.deferred){
+			await interaction.editReply({ content: 'There was an error while executing this command!', ephemeral: true });
```

Comment:
Why not `followUp()`?

Reply:
yea i think thats what i initially had in mind which is why I passed ephemeral, let me update it really quick